cmd/rollkit/main.go

```diff
@@ -19,8 +21,30 @@ func main() {
 		cmd.DocsGenCmd,
 		cmd.NewRunNodeCmd(),
 		cmd.VersionCmd,
+		cmd.NewTomlCmd(),
 	)
 
+	// In case there is a rollkit.toml file in the current dir or somewhere up the
+	// directory tree - we want to intercept the command and execute it against an entrypoint
+	// specified in the rollkit.toml file. In case of missing toml file or missing entrypoint key
+	// or missing actual entrypoint file - the normal rootCmd command is executed.
+	err := cmd.InterceptCommand(
+		rootCmd,
+		rollconf.ReadToml,
+		cmd.RunRollupEntrypoint,
+	)
+	switch {
+	case err == nil:
+		return
+	case errors.Is(err, cmd.ErrRunEntrypoint):
+		fmt.Printf("%v\n", err)
+		return
+	case errors.Is(err, rollconf.ErrReadToml):
+		fmt.Printf("%v\n", err)
+	case errors.Is(err, cmd.ErrRollkitCommand):
+	case errors.Is(err, cmd.ErrHelpVersion):
+	}
```

Comment:
If we update the InterceptCommand to return the boolean and error we can simplify this down as well. 
```suggestion
        fmt.Println("Checking for command intercept...")
	executed, err := cmd.InterceptCommand(
		rootCmd,
		rollconf.ReadToml,
		cmd.RunRollupEntrypoint,
	)
	if err != nil {
	        fmt.Println("Error intercepting Command: ",err)
	}
	if executed {
		return
	}	
         fmt.Println("Continuing without intercept....")
```